src/config/capabilityStatementResources.json

```diff
@@ -0,0 +1,267 @@
+{
+  "mode": "server",
+  "resource": [
+    {
+      "type" : "Library",
+      "profile" : "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/library-cqfm",
+      "supportedProfile" : [
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/computable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/publishable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/executable-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/modelinfo-library-cqfm",
+        "http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/quality-program-cqfm"
+      ],
+      "interaction" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "read",
+          "documentation" : "Read allows clients to get the definitions and details of repository libraries"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "code" : "search-type",
+          "documentation" : "Search allows clients to search for repository libraries"
+        }
+      ],
+      "searchParam" : [
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "url",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Library-url",
+          "type" : "uri"
+        },
+        {
+          "extension" : [
+            {
+              "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+              "valueCode" : "SHALL"
+            }
+          ],
+          "name" : "version",
+          "definition" : "http://hl7.org/fhir/SearchParameter/Library-version",
```

Comment:
The links for these definitions don't lead to a valid page for me. Is that something we need to address?

Reply:
Oh wow nice catch haha they also don't work for me. Not sure what it should be instead though?

Reply:
![mattsignal](https://user-images.githubusercontent.com/87094479/215769762-ac136967-38d7-40a7-af31-e821de0b9fee.png)